tests/Integration/TestCase.php

```diff
@@ -176,4 +176,46 @@ public function go_to_new_post(): int {
 
 		return $post_id;
 	}
+
+	/**
+	 * Simulate a single outbound HTTP request and response.
+	 *
```

Comment:
I think including docs for an example call in here would be useful to better understand how to use it.